assets/common/components/mlflow_model_local_validation/spec.yaml

```diff
@@ -1,7 +1,7 @@
 $schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
 
 name: mlflow_model_local_validation
-version: 0.0.6
+version: 0.0.7
```

Comment:
nit: move version change out of this PR, so you dont always have to sync with latest and also reduces any error in versioning